spectec/src/il2al/translate.ml

```diff
@@ -707,7 +722,7 @@ let rec kind_of_context e =
   | Il.CaseE ([[]; [{it = Il.Semicolon; _}]; []], e')
   | Il.ListE [ e' ]
   | Il.TupE [_ (* z *); e'] -> kind_of_context e'
-  | _ -> "Could not get kind_of_context" ^ Il.Print.string_of_exp e |> failwith
+  | _ -> error e.at "cannot get context of expression"
```

Comment:
Users will wonder what specifically is meant by "context", and why it wasn't possible to get it.

Reply:
I added more understandable terms, "frame" and "label" to express the reason.